### DISCUSSION TITLE
Lose header grid  Style with header fixed .net maui blazor

### DISCUSSION THREAD

**Discussion by walidmasa** [vikramlearning/blazorbootstrap#460] [Q&A]
i work with the component grid when the header fixed =false the header style is working fine and when i change the headerfixed=true the header lose the style (for more precision it loses the color of the backgrounds)

![Capture_with_HeaderFixed](https://github.com/vikramlearning/blazorbootstrap/assets/122237339/a8519898-6dca-4ecc-a8f6-15bf93c49554)
![Capture_without_HeaderFixed](https://github.com/vikramlearning/blazorbootstrap/assets/122237339/d4a25382-83af-476f-b4b7-b30c6a45ce7d)

**Comment by gvreddy04** [comment]
@walidmasa Thanks for trying **BlazorBootstrap**! Can you share a sample code with minimal steps to reproduce the issue?

Please also provide the following information:

.NET 6/7/8:
Server/WebAssembly/Web App (Interactive Render Mode - Web Assembly/Server):
Location (Per Page/Component or Global):
OS:
Browser:

**Comment by walidmasa** [comment]
thank you for coming back
I'm working on a .Net Maui Blazor project with .Net7.
here's a sample code:
-this is the main page code
MasterPage.razor
<div class="container-fluid gap-1 d-flex flex-column" style="display:flex;flex-direction:column; flex-grow: 1;min-height: 100%;">
    <div class="row">
        <div class="col-12">
            <EditFamilyArticle />
        </div>
    </div>
    <div class="row" style="flex-grow:1;">
        <div class="col-12">
            <GridViewFamilyArticles />
        </div>
    </div>
</div>
-this is the code for  GridViewFamilyArticles .razor 

<Grid @ref="gridFamilyArticle" TItem="FamilyArticleDto"
      Class="table table-hover table-bordered flex-fill gridAccess"
      DataProvider="FamilyArticleDataProvider"
      AllowFiltering="true"
      AllowSorting="true"
      AllowRowClick="true"
      OnRowDoubleClick="OnRowDoubleClick"
      Responsive="true" 
     FixedHeader="true"
      FiltersRowCssClass="bg-primary text-white bg-opacity-25 border-bottom-0"
      HeaderRowCssClass="bg-primary text-white border-bottom-0"
      access="true"
       id="@idGrid"
>

    <GridColumn  TItem="FamilyArticleDto" HeaderText="@Localizer[$"{idGrid}/CodeFamilyArticle"]" PropertyName="CodeFamilyArticle" SortKeySelector="item => item.CodeFamilyArticle">
        @context.CodeFamilyArticle
    </GridColumn>

</Grid>
